nginx/conf.d/default.conf

```diff
@@ -9,8 +9,8 @@ server {
   if (-d "/var/www/html/public") {
     set $docroot "html/public";
   }
-  if ($request_filename ~ /adminer) {
-    set $docroot "adminer";
+  if ($request_filename ~ /adminerevo) {
```

Comment:
should we also keep old url working? like redirecting /adminer to /adminverevo automatically

Reply:
that can be use ful, I'll make the change